src/tests/ftest/daos_test/suite.py

```diff
@@ -443,3 +443,19 @@ def test_daos_upgrade(self):
         :avocado: tags=daos_test,daos_core_test,test_daos_upgrade
         """
         self.run_subtest()
+
+    def test_daos_pipeline(self):
```

Comment:
You may need to add a tag here so the tests are skipped on release builds.  @phender do you know how to do this?

Reply:
well the test is not skipped just runs but the cmocka test function is skipped.